setup.py

```diff
@@ -59,6 +59,6 @@
     entry_points={
         "distutils.commands": ["check = readme_renderer.integration.distutils:Check"],
     },
-    extras_require={"md": "cmarkgfm>=0.5.0,<0.6.0"},
+    extras_require={"md": "cmarkgfm>=0.5.0"},
```

Comment:
```suggestion
    extras_require={"md": "cmarkgfm>=0.5.0,<0.7.0"},
```
Since there have been breaking changes between minor versions of `cmarkgfm` in the past (see #180) I'd prefer to keep an upper bound here.

Reply:
OK, done.